metrics/deploy/prometheus-ocs-rules.yaml

```diff
@@ -375,10 +375,10 @@ spec:
         runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/openshift-container-storage-operator/OSDCPULoadHigh.md
         severity_level: warning
       expr: |
-        pod:container_cpu_usage:sum{pod=~"rook-ceph-osd-.*"} / on(pod) kube_pod_resource_limit{resource='cpu',pod=~"rook-ceph-osd-.*"} > 0.80
-      for: 30m
+        pod:container_cpu_usage:sum{pod=~"rook-ceph-osd-.*"} / on(pod) kube_pod_resource_limit{resource='cpu',pod=~"rook-ceph-osd-.*"} > 0.90
+      for: 12h
       labels:
-        severity: warning
+        severity: info
```

Comment:
Couple of changes required:

1. Update the description and message to reflect the new threshold and "12h" period.
2. Update the runbook as well.
3. Update `severity_level` annotation.

Reply:
@weirdwiz did you update the runbook as well, don't see that change?

Reply:
I've updated the changes required here, runbook is not updated here so should be fine for this commit